modules/amf-registration/amf-registration-rest/amf-registration-rest-impl/src/main/java/com/liferay/amf/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -21,33 +17,34 @@
 public class AccountResourceImpl extends BaseAccountResourceImpl {
 
 	@Override
-	public Account getAccount(String accountId, Account account) throws Exception {
+	public Account getAccount(String accountId) throws Exception {
 
-		return _accountLocalService.getAccount(accountId, account.getFirstName(), account.getLastName(), account.getEmailAdress(),
-				account.getUserName(), account.getGenre(), account.getBirthday(), account.getBirthday(), account.getPassword(), account.getHomePhone(),
-				account.getMobilePhone(), account.getAdress1(),account.getAdress2(), account.getCity(), account.getState(), account.getSecurityQuestion(),
-				account.getSecurityAnswer(), account.getacceptedTou);
+		return (Account) _accountLocalService.getAccount(accountId);
```

Comment:
casting types might lead to errors. I would recommend to create a `mapper` class and include within it a `toDTO` method to convert `Account` (model) to `Account` (DTO).